rest/tests/test_rest_authorization.py

```diff
@@ -642,3 +648,234 @@ def test_members_delete(self):
             self.delete_member(urlfor='groups', repo_slug=self.repo.slug,
                                username=self.author_user.username,
                                group_type=group_type)
+
+    def test_resource_get(self):
+        """Test retrieve learning resource"""
+        self.assertEqual(
+            1, self.get_learning_resources(self.repo.slug)['count'])
+
+        tarball_file = self.get_course_single_tarball()
+        import_file(
+            tarball_file, self.repo.id, self.user.id)
+        resource = get_resources(self.repo.id).first()
+        resource.static_assets.add(StaticAsset.objects.first())
+        lr_id = self.get_learning_resources(self.repo.slug)['results'][0]['id']
```

Comment:
It looks like this is the ID of `resource` in the line above. If not, it's unclear what this is doing.